pkg/commands/command_helper.go

```diff
@@ -111,10 +115,11 @@ func (ch CommandHelper) PrintObj(obj runtime.Object) error {
 }
 
 func (ch CommandHelper) PrintChangeResult(change bool, format string, args ...interface{}) error {
+	if ch.dryRun {
```

Comment:
this change can result " (dry run) (no change)" and was intentionally avoided, since it is bit of repetition. `no change` captured both cases and was reflective of kubectl behaviour

Reply:
it looks like kubectl just says `(dry run)` which I think makes sense

```
$ k get pod some-pod -o yaml | k apply -f- --dry-run=client
> pod/some-pod configured (dry run)
```